### PR TITLE
Decode a Morse code message Solution

### PR DESCRIPTION
**Pull request by kelomo2502**
In this milestone:

- We create a method to decode a Morse code character,  which takes a string parameter, and return the corresponding character in uppercase 
- Created a method to decode an entire word in Morse code, which takes a string parameter, and returns the string representation.
- Created a method to decode the entire message in Morse code, which takes a string parameter, and returns the string representation. Every word will be separated by 3 spaces

### CODE REVIEW

**Review by TedLivist — COMMENTED** [review]
## STATUS: INVALID CODE REVIEW REQUEST 🛑 🛑

Hi @kelomo2502,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

### This is a collaborative project. Therefore ensure your partner contributes to this project

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: morse.rb [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  character = word.split
+  character.each { |c| decoder += decode_char(c) }
+  decoder
+end
+
+def decode_message(message)
+  decoder_message = ''
+  words = message.split(/   /)
+  words.each { |c| decoder_message += "#{decode_word(c)} " }
+  decoder_message
```
Comment (DammyShittu) [inline morse.rb]: - [ ] Your method for decoding a word returns white spaces at the end of the sentence. You can correct this by adding `#strip` to the returned message.

  ```
  ...
  decode_message.strip
  ```
  my-

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @kelomo2502 @sajid-munawar,

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by codepantha — APPROVED** [review]
# STATUS: APPROVED 🟢 

## Hi team 👋, 

Well done implementing the required changes 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 🎯 
✔️ Descriptive PR
✔️ No linter errors
✔️ Used GitFlow

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**